.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         run: npm ci
       - name: Install playground dependencies
         working-directory: src/playground
-        run: npm ci
+        run: npm install
```

Comment:
This is expected. npm ci will delete any existing node_modules folder and relies on the package-lock. json file to install the specific version of each package. It is significantly faster than npm install because it skips some features.

Reply:
`npm ci` works only if we have `package-lock.json`.